GERMLINE_0001.cpp

```diff
@@ -20,32 +20,73 @@ int MAX_ERR_HET = 1;
-		else if( strncmp(argv[i], "-to_snp", strlen("-to_snp")) == 0 && i < argc-1 )		rs_range[1] = argv[++i];
-		else if( strncmp(argv[i], "-print", strlen("-print")) == 0 )						PRINT_MATCH_HAPS = true;
-		else if( strncmp(argv[i], "-silent", strlen("-silent")) == 0 )						SILENT = true;
-		else if( strncmp(argv[i], "-debug", strlen("-debug")) == 0 )						DEBUG = true;
-		else if( strncmp(argv[i], "-map", strlen("-map")) == 0 && i < argc-1)				map = argv[++i];
-		else if( strncmp(argv[i], "-bits", strlen("-bits")) == 0 && i < argc-1)				MARKER_SET_SIZE = atoi(argv[++i]);
-		else if( strncmp(argv[i], "-homoz-only", strlen("-homoz-only")) == 0 )				{ ALLOW_HOM = true; HOM_ONLY = true; }
-		else if( strncmp(argv[i], "-homoz", strlen("-homoz")) == 0 )						ALLOW_HOM = true;
-		else if( strncmp(argv[i], "-bin_out", strlen("-bin_out")) == 0 )						BINARY_OUT = true;
-		else if( strncmp(argv[i], "-haploid", strlen("-haploid")) == 0 )					{ HAPLOID = true; HAP_EXT = true; }
-		else if( strncmp(argv[i], "-h_extend", strlen("-h_extend")) == 0 )					HAP_EXT = true;
-		else if( strncmp(argv[i], "-w_extend", strlen("-w_extend")) == 0 )					WIN_EXT = true;
+		if (strncmp(argv[i], "-min_m", strlen("-min_m")) == 0 && i < argc-1) {
+			MIN_MATCH_LEN = atof(argv[++i]);
+		}
+		else if (strncmp(argv[i], "-err_hom", strlen("-max_err")) == 0 && i < argc-1) {
+			MAX_ERR_HOM = atoi(argv[++i]);
+		}
+		else if (strncmp(argv[i], "-err_het", strlen("-max_err")) == 0 && i < argc-1) {
+			MAX_ERR_HET = atoi(argv[++i]);
+		}
+		else if (strncmp(argv[i], "-from_snp", strlen("-from_snp")) == 0 && i < argc-1 ) {
+			rs_range[0] = argv[++i];
+		}
+		else if (strncmp(argv[i], "-to_snp", strlen("-to_snp")) == 0 && i < argc-1 ) {
+			rs_range[1] = argv[++i];
+		}
+		else if (strncmp(argv[i], "-print", strlen("-print")) == 0 ) {
+			PRINT_MATCH_HAPS = true;
+		}
+		else if (strncmp(argv[i], "-silent", strlen("-silent")) == 0 ) {
+			SILENT = true;
+		}
+		else if (strncmp(argv[i], "-debug", strlen("-debug")) == 0 ) {
+			DEBUG = true;
+		}
+		else if (strncmp(argv[i], "-map", strlen("-map")) == 0 && i < argc-1) {
+			map = argv[++i];
+		}
+		else if (strncmp(argv[i], "-bits", strlen("-bits")) == 0 && i < argc-1) {
+			MARKER_SET_SIZE = atoi(argv[++i]);
+		}
+		else if (strncmp(argv[i], "-homoz-only", strlen("-homoz-only")) == 0 ) {
+			ALLOW_HOM = true;
+			HOM_ONLY = true;
+		}
+		else if (strncmp(argv[i], "-homoz", strlen("-homoz")) == 0 ) {
+			ALLOW_HOM = true;
+		}
+		else if (strncmp(argv[i], "-bin_out", strlen("-bin_out")) == 0 ) {
+			BINARY_OUT = true;
+		}
+		else if (strncmp(argv[i], "-haploid", strlen("-haploid")) == 0 ) {
+			HAPLOID = true;
+			HAP_EXT = true;
+		}
+		else if( strncmp(argv[i], "-h_extend", strlen("-h_extend")) == 0 ) {
+			HAP_EXT = true;
+		}
+		else if (strncmp(argv[i], "-w_extend", strlen("-w_extend")) == 0 ) {
+			WIN_EXT = true;
+		}
 		else if( strncmp(argv[i], "-version", strlen("-version")) == 0 ) {
 			cout << "version 1.5.2" << endl;
```

Comment:
Version bump here! Feels like v1.6 given the new features, per semantic versioning

Reply:
I actually add an -embark prefix also